### ISSUE TITLE
modReLU

### ISSUE THREAD

**Issue by lintongtong123** [CeeZh/CNN-based-on-Complex-Number#1]
您好，
我运行您的程序时，发现accuracy是在0.10左右，请问您实际运行时用的参数是多少呢，比如学习率、dropout用的KEEP_PROB？

**Comment by CeeZh** [comment]
抱歉，刚看到。
刚才直接用原来的参数跑一遍，准确率有0.8以上。你是直接运行，然后accuracy只有0.1吗？
原论文里是没有dropout的，我写的dropout是实部虚部一起drop的，本身应该没有问题，但是一旦设置KEEP_PROB不为1，拟合速度会减慢不少，甚至无法拟合。我觉得这可能需要从理论方面做解释，可能就是原论文里没有dropout的原因吧。
还有，我感觉复数网络不适合普通的图像。特别是本来就是实值矩阵表示的图像，需要先fft才能训练。这样不仅多此一举，而且准确率还会降低。

**Comment by zengjie617789** [comment]
@CeeZh 你好，想问一下，你这个程序与keras-complex程序复数网络层那块各自的处理方法好像都不太一样？您的代码清晰易懂，但我想问一下您数据split那块，在axis=-1进行split，可数据axis=-1不是维度为3 么？不是偶数能split么？是您在数据输入前进行了处理么？

**Comment by ShadowDSW** [comment]
Run source code without modification, accuracy on total training data equals 0.0999375
